pages/api/roadmap.ts

```diff
@@ -133,14 +134,16 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
         });
       }
     }
-    const toReturn = {
+
+    const toReturn: GithubIssueDataWithGroupAndChildren = {
       ...rootIssue,
       root_issue: true,
+      group: 'root',
       children
     };
 
     const data = {
-      ...addToChildren([{ children: [toReturn] }], {})[0].children[0],
+      ...addToChildren([toReturn], {} as IssueData)[0],
```

Comment:
I would love to see it taken the next step forward, but this is already much more maintainable.